docs/static/releasenotes.asciidoc

```diff
@@ -28,23 +28,67 @@ This section summarizes the changes in the following releases:
+ ** The introduction of an age retention policy, enabling the automatic removal of segments from a dead letter queue
+    based on the age of events within those segments.
+ ** We now expose further dead letter queue metrics on the monitoring API https://github.com/elastic/logstash/pull/14324[#14324]
+* Many AWS plugins are now bundled in a single integration plugin, enabling easier maintenance and upgrades of AWS based
+  plugins, which are now using version 3 of the AWS Ruby SDK.
+* Logstash now comes bundled with JDK17, while still providing compatibility with user-supplied JDK11.
+* Logstash is now available for download on M1 equipped MacOS devices, and come bundled with M1 native JDK17.
+
+
 
 [[notable-8.4.0]]
-==== Performance improvements and notable issues fixed
+==== Notable issues fixed
 
+* Only remove `/etc/systemd/system/logstash.service` when file is installed by Logstash https://github.com/elastic/logstash/pull/14200[#14200]
+* Fix Arcsight module compatibility with Elasticsearch `8.x` https://github.com/elastic/logstash/pull/13874[#13874]
+* Ensure timestamp values are serialized with a minimum of 3 decimal places to ensure millisecond precision timestamps match those from Logstash `7.x` https://github.com/elastic/logstash/pull/14299[#14299]
+* Fix issue with native Java plugin thread-safety and concurrency https://github.com/elastic/logstash/pull/14360[#14360]
+* Allow the ability to use Ruby codecs inside native Java plugins https://github.com/elastic/logstash/pull/13523[#13523]
 
-[[core-8.4.0]]
-==== Other changes to Logstash core
 
+[[dependencies-8.4.0]]
+==== Updates to dependencies
+
+* The bundled JDK has been updated to 17.0.4+8 https://github.com/elastic/logstash/pull/14427[#14427]
+* The version of Sinatra has been update to 2.2.2  https://github.com/elastic/logstash/pull/14454[#14454]
+* The version of Nokogiri has been update to 1.13.8 https://github.com/elastic/logstash/pull/14454[#14454]
 
 [[plugins-8.4.0]]
 ==== Plugin releases
 
+*Dead Letter Queue Input - 2.0.0*
+
+* Introduce the boolean `clean_consumed` setting to enable the automatic removal of completely consumed segments. Requires Logstash 8.4.0 or above https://github.com/logstash-plugins/logstash-input-dead_letter_queue/pull/43[#43]
+* Exposes metrics about segments and events cleaned by this plugin https://github.com/logstash-plugins/logstash-input-dead_letter_queue/pull/45[#45]
+
+*Xml Filter - 4.2.0*
+
+* Update Nokogiri dependency version https://github.com/logstash-plugins/logstash-filter-xml/pull/78[#78]
+
+* Aws Integration Plugin - 7.0.0*
+
+* Introduce AWS Integration Plugin. This is an integrated plugin, incorporating and replace the use of the following
+  individual plugins:
+  ** logstash-input-s3 (3.8.4)
+  ** logstash-input-sqs (3.3.2)
+  ** logstash-mixin-aws (5.1.0)
+  ** logstash-output-cloudwatch (3.0.10)
+  ** logstash-output-s3 (4.3.7)
+  ** logstash-output-sns (4.0.8)
+  ** logstash-output-sqs (6.0.0)
```

Comment:
Should we remove the old version numbers?

Reply:
Yes, I think so